### PR TITLE
Refactor gender to sex

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Refactor gender to sex to be more woke.

### Changed

- Refactor gender to sex

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b refactor-gender-to-sex -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/meta/clean/conftest.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def store_with_flow_cell_to_clean(
         has_backup=True,
     )
     sample: Sample = helpers.add_sample(
-        name=sample_id, internal_id=sample_id, sex="male", store=store, customer_id="cust500"
+        store=store, customer_id="cust500", internal_id=sample_id, name=sample_id, sex="male"
```
Comment (fevac) [inline tests/meta/clean/conftest.py:97]: replace "male" by constant?

File: tests/meta/clean/conftest.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def store_with_flow_cell_not_to_clean(
...
```
Comment (fevac) [inline tests/meta/clean/conftest.py:126]: as constant

File: tests/meta/orders/test_meta_orders_api.py [hunk 3]
```diff
@@ -348,10 +348,10 @@ def test_validate_sex_inconsistent_sex(
     for sample in order_data.samples:
         sample_obj: Sample = helpers.add_sample(
             store=store,
-            subject_id=sample.subject_id,
-            name=sample.name,
-            gender="male" if sample.sex == "female" else "female",
             customer_id=customer.internal_id,
+            sex="male" if sample.sex == "female" else "female",
```
Comment (fevac) [inline tests/meta/orders/test_meta_orders_api.py:352]: constants

File: tests/meta/orders/test_meta_orders_api.py [hunk 4]
```diff
@@ -409,10 +409,10 @@ def test_validate_sex_unknown_existing_sex(
     for sample in order_data.samples:
         sample_obj: Sample = helpers.add_sample(
             store=store,
-            subject_id=sample.subject_id,
-            name=sample.name,
-            gender="unknown",
             customer_id=customer.internal_id,
+            sex="unknown",
```
Comment (fevac) [inline tests/meta/orders/test_meta_orders_api.py:413]: constant

File: tests/meta/upload/conftest.py [hunk 5]
```diff
@@ -101,9 +101,7 @@ def mip_dna_case(mip_dna_context: CGConfig, helpers: StoreHelpers) -> Case:
         data_analysis=Pipeline.MIP_DNA,
     )
     dna_mip_sample: Sample = helpers.add_sample(
-        store=store,
-        internal_id="mip-dna-case",
-        application_type="wgs",
+        store=store, application_type="wgs", internal_id="mip-dna-case"
```
Comment (fevac) [inline tests/meta/upload/conftest.py:104]: application type as constant

**Review by fevac — APPROVED** [review]
Nice! 🌟 only few comments

File: tests/cli/workflow/conftest.py [hunk 6]
```diff
@@ -34,13 +34,13 @@ def analysis_store(base_store: Store, workflow_case_id: str, helpers: StoreHelpe
     case = helpers.add_case(_store, workflow_case_id, data_analysis=Pipeline.MIP_DNA)
 
     dna_sample = helpers.add_sample(
-        _store, "dna_sample", is_rna=False, last_sequenced_at=datetime.now(), reads=10000000
+        _store, "dna_sample", is_rna=False, reads=10000000, last_sequenced_at=datetime.now()
```
Comment (islean) [inline tests/cli/workflow/conftest.py:37]: why do we reshuffle so many parameters?

File: tests/conftest.py [hunk 7]
```diff
@@ -3378,7 +3378,7 @@ def store_with_sequencing_metrics(
     ]
     helpers.add_flow_cell(store=store, flow_cell_name=flow_cell_name)
     helpers.add_sample(
-        name=sample_id, internal_id=sample_id, sex="male", store=store, customer_id="cust500"
+        store=store, customer_id="cust500", internal_id=sample_id, name=sample_id, sex="male"
```
Comment (islean) [inline tests/conftest.py:3381]: Constant for male

File: tests/meta/workflow/test_balsamic.py [hunk 8]
```diff
@@ -51,13 +51,12 @@ def test_get_verified_gender_unknown(caplog):
...
-    retrieved_gender: Gender = BalsamicAnalysisAPI.get_verified_gender(sample_obj)
+    retrieved_gender: Sex = BalsamicAnalysisAPI.get_verified_gender(sample_obj)
 
     # THEN gender must match the expected one
-    assert retrieved_gender == Gender.FEMALE
+    assert retrieved_gender == Sex.FEMALE
     assert (
-        f"The provided gender is unknown, setting {Gender.FEMALE.value} as the default"
-        in caplog.text
+        f"The provided gender is unknown, setting {Sex.FEMALE.value} as the default" in caplog.text
```
Comment (islean) [inline tests/meta/workflow/test_balsamic.py:59]:
```suggestion
        f"The provided sex is unknown, setting {Sex.FEMALE.value} as the default" in caplog.text
```

**Review by islean — APPROVED** [review]
Looks good! Some minor comments

File: cg/meta/workflow/balsamic.py [hunk 9]
```diff
@@ -285,13 +285,13 @@ def get_verified_gender(sample_data: dict) -> str:
         gender = next(iter(sample_data.values()))["gender"]
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply cg/meta/workflow/balsamic.py:285]: I think so

File: cg/models/report/validators.py [hunk 10]
```diff
@@ -57,7 +57,7 @@ def get_path_as_string(file_path: str | None) -> str:
     return Path(file_path).name if file_path and Path(file_path).is_file() else NA_FIELD
 
 
-def get_gender_as_string(gender: Gender | None) -> str:
+def get_gender_as_string(gender: Sex | None) -> str:
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply cg/models/report/validators.py:60]: Another PR

File: tests/cli/workflow/conftest.py (hunk 6 above)
Comment (henrikstranneheim) [reply tests/cli/workflow/conftest.py:37]: I guess creating test cases. Should be refactored in another PR./

File: tests/meta/clean/conftest.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/meta/clean/conftest.py]: Fixed

File: tests/meta/orders/test_meta_orders_api.py (hunk 3 above)
Comment (henrikstranneheim) [reply tests/meta/orders/test_meta_orders_api.py]: Fixed

File: tests/meta/upload/conftest.py (hunk 5 above)
Comment (henrikstranneheim) [reply tests/meta/upload/conftest.py:104]: Another PR

File: tests/meta/orders/test_meta_orders_api.py (hunk 4 above)
Comment (henrikstranneheim) [reply tests/meta/orders/test_meta_orders_api.py]: Fixed

File: tests/meta/clean/conftest.py (hunk 2 above)
Comment (henrikstranneheim) [reply tests/meta/clean/conftest.py]: Fixed

File: cg/meta/workflow/balsamic.py [hunk 11]
```diff
@@ -285,13 +285,13 @@ def get_verified_gender(sample_data: dict) -> str:
...
 
         if all(val["gender"] == gender for val in sample_data.values()) and gender in set(
-            value for value in Gender
+            value for value in Sex
         ):
-            if gender not in [Gender.FEMALE, Gender.MALE]:
+            if gender not in [Sex.FEMALE, Sex.MALE]:
                 LOG.warning(
-                    f"The provided gender is unknown, setting {Gender.FEMALE.value} as the default"
+                    f"The provided gender is unknown, setting {Sex.FEMALE.value} as the default"
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply cg/meta/workflow/balsamic.py]: Fixed

File: tests/apps/scout/test_scout_models.py [hunk 12]
```diff
@@ -177,10 +177,10 @@ def test_set_gender_if_provided():
     """Test to validate that the gender is not altered when set and not Gender.OTHER."""
 
     # GIVEN a gender which is not Gender.OTHER as input
-    gender: PlinkGender = PlinkGender.FEMALE
+    gender: PlinkSex = PlinkSex.FEMALE
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply tests/apps/scout/test_scout_models.py]: Fixed

File: tests/cli/add/test_cli_add_relationship.py [hunk 13]
```diff
@@ -193,7 +193,7 @@ def test_add_relationship_father(
     sample = helpers.add_sample(disk_store)
     sample_id = sample.internal_id
 
-    father = helpers.add_sample(disk_store, name="father", gender="male")
+    father = helpers.add_sample(disk_store, sex="male", name="father")
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply tests/cli/add/test_cli_add_relationship.py]: Fixed

File: tests/conftest.py (hunk 7 above)
Comment (henrikstranneheim) [reply tests/conftest.py]: Fixed

File: tests/meta/workflow/test_balsamic.py (hunk 8 above)
Comment (henrikstranneheim) [reply tests/meta/workflow/test_balsamic.py]: Fixed

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

**Comment by henrikstranneheim** [comment]
### cg upload create-scout-load-config justhusky: 👍 
```
θ62° [henrik.stranneheim@hasta:~] [S_base] 20s $ cg upload create-scout-load-config justhusky
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
----------------- UPLOAD -----------------
Fetching family object
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
----------------- CREATE CONFIG -----------------------
Create load config
Generate scout load config
Found pipeline mip-dna
Generate load config for mip case
Including cohorts to scout load config
Adding phenotype groups to scout load config
Adding phenotype terms to scout load config
Including MIP specific case level files
Get file with tags {'peddy', 'ped-check'}
Get file with tags {'ped', 'peddy'}
Get file with tags {'sex-check', 'peddy'}
Get file with tags {'smn-calling'}
Get file with tags {'mobile-elements', 'vcf', 'clinical'}
Get file with tags {'mobile-elements', 'vcf', 'research'}
Get file with tags {'vcf-snv-clinical'}
Get file with tags {'vcf-snv-research'}
Get file with tags {'vcf-str'}
Get file with tags {'vcf-sv-clinical'}
Get file with tags {'vcf-sv-research'}
Include MultiQC report to case
Get file with tags {'multiqc-html'}
Include delivery report to case
Get file with tags {'delivery-report'}
Could not find any files matching the tags {'delivery-report'}
Building samples
Building sample earlycasualcaiman
Could not fetch sample earlycasualcaiman from LIMS: 404: Sample not found: earlycasualcaiman
Adding common files for sample earlycasualcaiman
Get file with tags {'earlycasualcaiman', 'bam'}
Get file with tags {'earlycasualcaiman', 'cram'}
Including MIP specific sample level files
Get file with tags {'earlycasualcaiman', 'vcf2cytosure'}
Get file with tags {'earlycasualcaiman', 'bam-mt'}
Get file with tags {'chromograph', 'earlycasualcaiman', 'autozyg'}
Get file with tags {'chromograph', 'earlycasualcaiman', 'tcov'}
Get file with tags {'chromograph', 'regions', 'earlycasualcaiman'}
Could not find any files matching the tags {'chromograph', 'regions', 'earlycasualcaiman'}
Get file with tags {'chromograph', 'earlycasualcaiman', 'sites'}
Could not find any files matching the tags {'chromograph', 'earlycasualcaiman', 'sites'}
Get file with tags {'expansionhunter', 'bam', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'earlycasualcaiman', 'bam-index'}
Could not find any files matching the tags {'expansionhunter', 'earlycasualcaiman', 'bam-index'}
Get file with tags {'expansionhunter', 'vcf-str', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'earlycasualcaiman', 'mitodel'}
Building sample slowlycivilbuck
Could not fetch sample slowlycivilbuck from LIMS: 404: Sample not found: slowlycivilbuck
Adding common files for sample slowlycivilbuck
Get file with tags {'bam', 'slowlycivilbuck'}
Get file with tags {'slowlycivilbuck', 'cram'}
Including MIP specific sample level files
Get file with tags {'slowlycivilbuck', 'vcf2cytosure'}
Get file with tags {'bam-mt', 'slowlycivilbuck'}
Get file with tags {'chromograph', 'slowlycivilbuck', 'autozyg'}
Get file with tags {'chromograph', 'slowlycivilbuck', 'tcov'}
Get file with tags {'chromograph', 'regions', 'slowlycivilbuck'}
Could not find any files matching the tags {'chromograph', 'regions', 'slowlycivilbuck'}
Get file with tags {'chromograph', 'slowlycivilbuck', 'sites'}
Could not find any files matching the tags {'chromograph', 'slowlycivilbuck', 'sites'}
Get file with tags {'expansionhunter', 'bam', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'slowlycivilbuck', 'bam-index'}
Could not find any files matching the tags {'expansionhunter', 'slowlycivilbuck', 'bam-index'}
Get file with tags {'expansionhunter', 'vcf-str', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'slowlycivilbuck', 'mitodel'}
Building sample hugelymodelbat
Could not fetch sample hugelymodelbat from LIMS: 404: Sample not found: hugelymodelbat
Adding common files for sample hugelymodelbat
Get file with tags {'hugelymodelbat', 'bam'}
Get file with tags {'hugelymodelbat', 'cram'}
Including MIP specific sample level files
Get file with tags {'hugelymodelbat', 'vcf2cytosure'}
Get file with tags {'hugelymodelbat', 'bam-mt'}
Get file with tags {'chromograph', 'hugelymodelbat', 'autozyg'}
Get file with tags {'chromograph', 'hugelymodelbat', 'tcov'}
Get file with tags {'chromograph', 'regions', 'hugelymodelbat'}
Get file with tags {'chromograph', 'hugelymodelbat', 'sites'}
Get file with tags {'expansionhunter', 'bam', 'hugelymodelbat'}
Get file with tags {'expansionhunter', 'hugelymodelbat', 'bam-index'}
Could not find any files matching the tags {'expansionhunter', 'hugelymodelbat', 'bam-index'}
Get file with tags {'expansionhunter', 'hugelymodelbat', 'vcf-str'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'hugelymodelbat', 'mitodel'}
Generate madeline output to /home/henrik.stranneheim/tmp/tmp0p6o41eb/madeline.xml
Generating madeline input lines
Running command /home/proj/stage/servers/resources/hasta.scilifelab.se/madeline2 --color --nolabeltruncation --outputprefix /home/henrik.stranneheim/tmp/tmp0p6o41eb/madeline /home/henrik.stranneheim/tmp/tmp6z9ww51h
Found load config owner='cust000' family='justhusky' family_name='SWEDAC-2019-HG-05-Trio-NovaSeq' synopsis=None phenotype_terms=None phenotype_groups=None gene_panels=['BRAIN', 'Cardiology', 'CILM', 'CH', 'CTD', 'DIAB', 'ENDO', 'EP', 'HEARING', 'HYDRO', 'IBMFS', 'IEM', 'IF', 'mcarta', 'MHT', 'MIT', 'MOVE', 'mtDNA', 'NBS-M', 'NEURODEG', 'NMD', 'OMIM-AUTO', 'PANELAPP-GREEN', 'PEDHEP', 'PID', 'PIDCAD', 'RETINA', 'SKD', 'SOVM', 'STROKE', 'AID', 'Inherited cancer'] default_gene_panels=['OMIM-AUTO'] cohorts=None human_genome_build='37' rank_model_version='1.36' rank_score_threshold=5 sv_rank_model_version='1.11' analysis_date=datetime.datetime(2023, 12, 20, 9, 46, 14) samples=[ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/earlycasualcaiman_lanes_1_sorted_md.cram', rna_alignment_path=None, analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='0', mother='0', phenotype='unaffected', sample_id='earlycasualcaiman', sample_name='ACC5339A2--300M', sex='male', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/earlycasualcaiman_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb_rhocall_norm_af_rhoviz.earlycasualcaiman.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/earlycasualcaiman_lanes_1_sorted_md_tcov_', upd_regions=None, upd_sites=None), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_earlycasualcaiman.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_earlycasualcaiman.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_cyto.earlycasualcaiman.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/earlycasualcaiman_lanes_1_sorted_md_mitodel.txt'), ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/slowlycivilbuck_lanes_1_sorted_md.cram', rna_alignment_path=None, analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='0', mother='0', phenotype='unaffected', sample_id='slowlycivilbuck', sample_name='ACC5339A3--300M', sex='female', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/slowlycivilbuck_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb_rhocall_norm_af_rhoviz.slowlycivilbuck.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/slowlycivilbuck_lanes_1_sorted_md_tcov_', upd_regions=None, upd_sites=None), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_slowlycivilbuck.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_slowlycivilbuck.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_cyto.slowlycivilbuck.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/slowlycivilbuck_lanes_1_sorted_md_mitodel.txt'), ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_lanes_1_sorted_md.cram', rna_alignment_path=None, analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='earlycasualcaiman', mother='slowlycivilbuck', phenotype='affected', sample_id='hugelymodelbat', sample_name='ACC5339A4--300M', sex='male', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb_rhocall_norm_af_rhoviz.hugelymodelbat.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_lanes_1_sorted_md_tcov_', upd_regions='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_gatkcomb_rhocall_norm_af_upd.regions_', upd_sites='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_gatkcomb_rhocall_norm_af_upd.sites_'), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_hugelymodelbat.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun_hugelymodelbat.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_cyto.hugelymodelbat.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/hugelymodelbat_lanes_1_sorted_md_mitodel.txt')] delivery_report=None coverage_qc_report=None cnv_report=None multiqc='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/multiqc_report.html' track='rare' chromograph_image_files=None chromograph_prefixes=None madeline='/home/henrik.stranneheim/tmp/tmp0p6o41eb/madeline.xml' peddy_check='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb.ped_check.csv' peddy_ped='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb.peddy.ped' peddy_sex='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb.sex_check.csv' smn_tsv='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_smn.tsv' variant_catalog=None vcf_mei='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_me_ann_vep_filter.selected.vcf.gz' vcf_mei_research='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_me_ann_vep_filter.vcf.gz' vcf_snv='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb_rhocall_norm_af_mt_frqf_cadd_vep_parsed_ranked.selected.vcf.gz' vcf_snv_research='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_gatkcomb_rhocall_norm_af_mt_frqf_cadd_vep_parsed_ranked.vcf.gz' vcf_str='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_exphun.vcf.gz' vcf_sv='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_comb_ann_vep_parsed_ranked.selected.vcf.gz' vcf_sv_research='/home/proj/stage/housekeeper-bundles/justhusky/2023-12-18/justhusky_comb_ann_vep_parsed_ranked.vcf.gz'
```

### cg workflow mip-dna start justhusky --mip-dry-run 👍 
```
leting analysis: 282899, for case: justhusky
θ60° [henrik.stranneheim@hasta:~] [S_base] 6s $ cg workflow mip-dna start justhusky --mip-dry-run
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Case justhusky exists in Status DB
Starting full MIP analysis workflow for case justhusky
Flow cell check is not applicable - the case is either down sampled or external.
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R1_001.fastq.gz
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R2_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R1_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R2_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R1_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R2_001.fastq.gz
Case justhusky exists in Status DB
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/earlycasualcaiman/fastq/1_171015_HHT5NDSXX_earlycasualcaiman_XXXXXX_1.fastq.gz
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/earlycasualcaiman/fastq/1_171015_HHT5NDSXX_earlycasualcaiman_XXXXXX_2.fastq.gz
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/slowlycivilbuck/fastq/1_171015_HHT5NDSXX_slowlycivilbuck_XXXXXX_1.fastq.gz
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/slowlycivilbuck/fastq/1_171015_HHT5NDSXX_slowlycivilbuck_XXXXXX_2.fastq.gz
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/hugelymodelbat/fastq/1_171015_HHT5NDSXX_hugelymodelbat_XXXXXX_1.fastq.gz
Destination path already exists: /home/proj/stage/rare-disease/cases/justhusky/wgs/hugelymodelbat/fastq/1_171015_HHT5NDSXX_hugelymodelbat_XXXXXX_2.fastq.gz
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export panel --bed BRAIN Cardiology CILM CH CTD DIAB ENDO EP HEARING HYDRO IBMFS IEM IF mcarta MHT MIT MOVE mtDNA NBS-M NEURODEG NMD OMIM-AUTO PANELAPP-GREEN PEDHEP PID PIDCAD RETINA SKD SOVM STROKE AID Inherited cancer --build 37
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export managed --build 37
Case justhusky exists in Status DB
Config file saved to /home/proj/stage/rare-disease/cases/justhusky/pedigree.yaml
Case justhusky exists in Status DB
earlycasualcaiman: downsampled/external - skip evaluation
Running command /home/proj/stage/bin/miniconda3/bin/conda run --name S_mip12.0 /home/proj/stage/bin/miniconda3/envs/S_mip12.0/bin/mip analyse rd_dna --config /home/proj/stage/servers/config/hasta.scilifelab.se/mip12.0-dna-stage.yaml justhusky --slurm_quality_of_service high --email henrik.stranneheim@scilifelab.se --dry_run_all --qccollect_skip_evaluation

[henrik.stranneheim@hasta:~] [S_base] 1m39s $ cat /home/proj/stage/rare-disease/cases/justhusky/pedigree.yaml
---
case: justhusky
default_gene_panels:
- OMIM-AUTO
samples:
- analysis_type: wgs
  capture_kit: twistexomerefseq_9.1_hg19_design.bed
  expected_coverage: 1
  father: '0'
  mother: '0'
  phenotype: unaffected
  sample_display_name: ACC5339A2--300M
  sample_id: earlycasualcaiman
  sex: male
- analysis_type: wgs
  capture_kit: twistexomerefseq_9.1_hg19_design.bed
  expected_coverage: 1
  father: '0'
  mother: '0'
  phenotype: unaffected
  sample_display_name: ACC5339A3--300M
  sample_id: slowlycivilbuck
  sex: female
- analysis_type: wgs
  capture_kit: twistexomerefseq_9.1_hg19_design.bed
  expected_coverage: 1
  father: earlycasualcaiman
  mother: slowlycivilbuck
  phenotype: affected
  sample_display_name: ACC5339A4--300M
  sample_id: hugelymodelbat
  sex: male
```

**Comment by henrikstranneheim** [comment]
Deployed to production:

```
Log deploy... done.
cg, version 54.5.3
```

**Pull request merged**